static/templates/message_body.hbs

```diff
@@ -48,6 +35,15 @@
     {{/unless}}
 {{/unless}}
 
+{{> message_controls}}
+
+<a href="{{ msg/url }}" class="message_time{{#if msg.locally_echoed}} notvisible{{/if}}{{#if status_message}} status-time{{/if}}">
+    {{#unless include_sender}}
+    <span class="copy-paste-text">&nbsp;</span>
+    {{/unless}}
+    {{timestr}}
+</a>
```

Comment:
Here, we are reordering the timestamp to be after the message body within the message in the HTML. Is this required? I think we intentionally had the timestamp before the message body for copy/paste reasons, but trying to test, it kinda looks like we completely broken being able to copy/paste messages including the timestamps at some point? Or maybe there's some decision I've forgotten; @evykassirer would know given their work in the area.

So maybe there's an unrelated regression here, but I'm curious whether this change in order will make our life difficult for fixing the above bug, and whether there's a way we can avoid it with a few extra lines of CSS.

Reply:
> Here, we are reordering the timestamp to be after the message body within the message in the HTML. Is this required?

No, I reverted them back. I might have been trying something here that I forgot.